packages/community/lib/extensions/jsx/WalletConnect.tsx

```diff
@@ -0,0 +1,15 @@
+export const WalletConnect = () => (
+    <svg 
+        fill="none" 
+        height="332"
+        viewBox="0 0 480 332" 
+        width="480" 
```

Comment:
Let's give apps the opportunity to size the SVG as they please. 
```suggestion
        viewBox="0 0 480 332" 
```